picobot/handlers.py

```diff
@@ -311,6 +311,55 @@ def del_sticker(bot: Bot, update: Update):
         msg.reply_text(responses.REMOVE_STICKER_HELP)
 
 
+def change_emojis(bot: Bot, update: Update):
+    msg: Message = update.message
+    msg_type = get_msg_type(msg)
+    user_id = msg.from_user.id
+    response = responses.ERROR_MSG
+    splittext = shlex.split(msg.text)
+
+    # remove sticker from pack
+    try:
+        if msg_type == MsgType.REP_STICKER:
+            pack_name = msg.reply_to_message.sticker.set_name
+            sticker_id = msg.reply_to_message.sticker.file_id
+
+            if pack_name is None:
+                msg.reply_text('Não é possível remover o sticker de um pack inexistente.')
+                return
```

Comment:
Faltou dizer o que acontece quando a mensagem não é um reply para um sticker. Nesse caso, daria para tratar isso antes de tudo:

```suggestion
        if msg_type != MsgType.REP_STICKER:
            msg.reply_text(responses.CHANGE_EMOJIS_HELP)
            return

        pack_name = msg.reply_to_message.sticker.set_name
        sticker_id = msg.reply_to_message.sticker.file_id

        if pack_name is None:
            msg.reply_text(responses.PACK_DOES_NOT_EXIST)
            return
```

Outra coisa é usar o [responses](https://github.com/caravelahc/pico-bot/blob/master/picobot/responses.py) para as mensagens de resposta do bot. Coloquei já na sugestão como seria o uso delas, só faltaria adicionar o `CHANGE_EMOJIS_HELP` e o `PACK_DOES_NOT_EXIST` no `responses.py`.